sky/unit/test/widget/stack_test.dart

```diff
@@ -68,18 +71,20 @@ void main() {
       tester.pumpWidget(new Stack([ new Positioned(left: 10.0, child: container) ]));
       Element containerElement = tester.findElementByKey(key);
 
-      expect(containerElement.renderObject.parentData.top, isNull);
-      expect(containerElement.renderObject.parentData.right, isNull);
-      expect(containerElement.renderObject.parentData.bottom, isNull);
-      expect(containerElement.renderObject.parentData.left, equals(10.0));
+      StackParentData parentData = containerElement.renderObject.parentData;
```

Comment:
declare it earlier so that the code here has the same indentation as the later use